addons/web_editor/static/src/js/editor/rte.js

```diff
@@ -507,6 +507,15 @@ var RTEWidget = Widget.extend({
      * @param {jQuery} $editable
      */
     _enableEditableArea: function ($editable) {
+        if ($editable.data('oe-type') === "datetime" || $editable.data('oe-type') === "date") {
+            debugger;
+            var $linkedFieldNodes = $('#wrapwrap [data-oe-id="' + $editable.data('oe-id') + '"][data-oe-field="' + $editable.data('oe-field') + '"][data-oe-model="' + $editable.data('oe-model') + '"]').not($editable);
+            $linkedFieldNodes.addClass('o_editable_date_field_linked');
```

Comment:
The role of this class seems like a big improvement not a fix, or maybe I missed something? (smaller diff the better because of a fix but also because you will have a hard time forward-porting this otherwise)